recipes/fairease-source/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+$PYTHON -m pip install --no-deps .
+
+# Copy/link executables somewhere on the path
+python_path=$(find $PREFIX -name SOURCE)
```

Comment:
Is there an executable called SOURCE? Its not part of the tests.

Reply:
No, this is to parse the part of the path (/a/b/c/SOURCE/d/e.py) following "SOURCE".